nsv13/code/game/gamemodes/overmap/objectives/board_ship.dm

```diff
@@ -11,6 +11,7 @@
 	var/ship_type = pick(GLOB.boardable_ship_types)
 	target_ship = instance_overmap(ship_type)
 	target_ship.block_deletion = TRUE
+	target_ship.hammerlocked = TRUE
```

Comment:
I might suggest adding this to ai_load_interior, actually - then both the boarding objective and crew-locked ships will be covered